docs/contributing-guide.md

```diff
@@ -139,6 +139,18 @@ local Python script to override your installation in `$HOME/.bloop`:
 1. Give executable permission to the install script: `chmod +x $BLOOP_REPO/frontend/target/install.py`.
 1. Run script with `./$BLOOP_REPO/frontend/target/install.py`.
 
+You will something like this:
```

Comment:
```suggestion
You will see something like this:
```

Reply:
I must have been pretty hungry while writing this. Thanks for spotting it!